pydis_site/README.md

```diff
@@ -0,0 +1,52 @@
+# `pydis_site` project directory
+
+This directory hosts the root of our **Django project**[^1], and is responsible
+for all logic powering our website. Let's go over the directories in detail:
+
+- [`apps`](./apps) contains our **Django apps**. If you want to add your own API
+  endpoint or new functionality to our homepage, that's the place to go.
+  <!-- Each individual application also has its own README.md that you can click
+  through. -->
+
+- [`static`](./static) contains our **static files**, such as CSS, JavaScript,
+  images, and anything else that isn't either content or Python code. Static
+  files relevant for a specific application are put into subdirectories named
+  after the application.
+
+- [`templates`](./templates) contains our **Django templates**. Like with static
+  files, templates specific to a single application are stored in a subdirectory
+  named after that application. We also have two special templates here:
+
+  - `404.html`, which is our error page shown when a site was not found.
+
+  - `500.html`, which is our error page shown in the astronomically rare case
```

Comment:
Citation needed

Reply:
hahaha. this is fine. I like the boldness.